sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -214,7 +214,7 @@
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<ConfigurationSetting>> setConfigurationSettingWithResponse(ConfigurationSetting setting,
-                                                                       boolean ifUnchanged) {
+                                                                                    boolean ifUnchanged) {
```

Comment:
This is squashed to the right margin, guidance would be to do an 8 space indentation rather than aligning parameters.

Reply:
The same applies to the other methods below. It makes it really hard to read.

https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html#248

> If the above rules lead to confusing code or to code that's squished up against the right margin, just indent 8 spaces instead.